app/src/main/java/com/github/sdp/ratemyepfl/viewmodel/AddReviewViewModel.kt

```diff
@@ -17,13 +23,24 @@ import javax.inject.Inject
  */
 @HiltViewModel
 class AddReviewViewModel @Inject constructor(
-    private val database: ReviewsRepositoryInterface
+    private val database: ReviewsRepositoryInterface,
```

Comment:
It may be clearer to rename it as `reviewRepo`, also to match the name structure of the next attribute.

Reply:
Also change the names in `ReviewViewModel` to match the ones

Reply:
Yeah it will be clearer, I fixed it.